AgentDiscoveries-Frontend/app/src/components/search-forms/search-result.jsx

```diff
@@ -24,10 +25,25 @@ export default class SearchResult extends React.Component {
 
     renderResultBody(result) {
         return Object.keys(result).map(key => {
+            if(this.isADate(result[key])){
+                const time = moment(result[key]).format('H:mm:ss');
+
+                return (
+                        <div>
+                        <p key ={key} id={key}>{`${key}: ${result[key]}`}</p>
+                        <p key={key}> timeInSubmittersTimeZone: {time} </p>
+                        </div>
+                );
+            }
             return <p key={key} id={key}>{`${key}: ${result[key]}`}</p>;
         });
     }
 
+    isADate(date){
+        date = moment(date)
+        return (date.isValid() && (date.year() !== 1970))
```

Comment:
I can that the `date.year() !== 1970` works as a failsafe for dodgy dates being accepted, but it's a bit of a hack - it depends on which library you're using for date parsing for example. The isValid method looks much more relevant/honest - can we adjust so that we can just use that?